pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -233,13 +225,13 @@ let
           maintainers.thoughtpolice
         ];
         platforms = platforms.linux;
-      } // extraMeta;
-    };
+      };
+    } // extraMeta;
```

Comment:
This doesn't make sense to me. Why shouldn't `extraMeta` be merged with the existing `meta` attribute?